src/odb/include/odb/db.h

```diff
@@ -7870,6 +7872,10 @@ class dbModule : public dbObject
   // Adding an inst to a new module will remove it from its previous
   // module.
   void addInst(dbInst* inst);
+  // For performance look up
+  void addInstToHash(dbInst* inst);
+  void addInstToHash(dbModInst* inst);
+  void addModBTermToHash(dbModBTerm* modbterm);
```

Comment:
I don't like that callers are managing the internal hashes.

Reply:
agreed. Removed. Hashes populated/depopulated by very low level db calls only (eg create/destroy/stream).